woocommerce/woocommerce
# Update concurrency group for pr label workflow

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Updated the concurrency group for the PR labeller workflow to something unique to the PR. For `pull_request_target` trigger the `github.ref` [value is the base branch](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target), making the group name for PRs from the same base branch identical. The effect is that 'Label Pull Request Project' running on one PR could cancel the same workflow running for another PR. 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Need two PRs for branches with same base branch.
2. Update one branch so that 'Label Pull Request Project' starts.
3. Update the second branch so that 'Label Pull Request Project' starts in parallel to the previous one.

The first workflow should not be cancelled.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
